Migrate SwerveLib to ServoLib and add LCD Battery Menu

kirstenalice opened the pull request.

Commit:
swerveLib.c now implements the servo library

Commit:
Eugene.h --> SwerveLib.h

Commit:
implemented servo AND added battery voltage on LCD

matticoli commented:
Remove battery print statements from opcontrol.c

Lgtm other than that, will need to test before merge in case something in ServoLib is broken. 

We may need to mess with the PE battery conversion factor once we test on the bot (different edition)

Commit:
Kirsten's PRE-SBHS changes

swerveLib.c line 346 expected declaration or statement at the end of
input

THINGS TO DO:
delete horizontal compnent on left joystick to keep in tank

set LR and RR to 0

a lift potentiometer is going to be added into port 1 or 8

LIFT USES SERVOLIB
